grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -470,7 +472,6 @@ private void useRoundRobinLists(
         final Subchannel subchannel;
         if (newBackendAddrList.isEmpty()) {
           if (subchannels.size() == 1) {
-            cancelFallbackTimer();
```

Comment:
What's going on here? I'm having trouble figuring out why the timer cancellation was there to begin with and why it is being removed.

Reply:
I was surprised by having a timer cancellation there as well. It was introduced by #6734. I guess Jihun just didn't look deep enough to the entire implementation but just intended to fix the immediate issue (and I, as the reviewer wasn't aware of that as well 😢 ). And that didn't cause problem. But really, this shouldn't be there.

Reply:
SGTM.